index.js

```diff
@@ -4,7 +4,7 @@ var path = require('path');
 
 var commentRx = /^\s*\/(?:\/|\*)[@#]\s+sourceMappingURL=data:(?:application|text)\/json;(?:charset[:=]\S+;)?base64,(.*)$/mg;
 var mapFileCommentRx =
-  // //# sourceMappingURL=foo.js.map                       /*# sourceMappingURL=foo.js.map */
+  /*# sourceMappingURL=foo.js.map */
```

Comment:
Can't see the regex anymore since `//` was entirely removed. Could you please keep them in and escape them instead, i.e. `\/\/` and put a comment above that it was escaped due to a Safari bug.